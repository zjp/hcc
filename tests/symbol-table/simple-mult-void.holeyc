bool glob;
void glob;
int main(){
	int a;
	glob = a;
}
