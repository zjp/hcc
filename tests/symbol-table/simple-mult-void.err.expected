FATAL [2,1]: Invalid type in declaration
FATAL [2,6]: Multiply declared identifier
Name Analysis Failed
